be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -26,6 +26,8 @@
 #include "vec/exprs/vexpr_context.h"
 #include "vec/utils/util.hpp"
 
+#define PREFETCH_COUNT (size_t(32 * 1024) / CACHE_LINE_SIZE)
```

Comment:
To achieve best performance, we should take more information about cache size.
For example, l1d cache size is 64KB in AMD Epyc server processor[1] while 48KB in Intel Icelake one[2]. 

[1] https://www.club386.com/amd-unleashes-epyc-7003-series-with-3d-v-cache
[2] https://www.techpowerup.com/248825/intel-increases-l1d-and-l2-cache-sizes-with-ice-lake